pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -109,6 +109,7 @@ public SuccessResponse deleteSchema(
     return new SuccessResponse("Schema " + schemaName + " deleted");
   }
 
+  @Deprecated
```

Comment:
I am not sure if we can deprecate this, as I believe swagger UI uses this path (see https://pinot.readthedocs.io/en/latest/management_api.html#pinot-management-console).

Reply:
The fact that it is deprecated does not mean that its functionality cannot be used. This method can still be used in places such as script creation schema, but it is suggested that this method cannot be used in controller web. Some tests will be added later

Reply:
The fact that it is deprecated does not mean that its functionality cannot be used. This method can still be used in places such as script creation schema. it is suggested that this method cannot be used in controller web.